AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/AWSIncomingEventReconciliationQueue.swift

```diff
@@ -82,13 +83,15 @@ final class AWSIncomingEventReconciliationQueue: IncomingEventReconciliationQueu
         eventReconciliationQueueTopic.send(.paused)
     }
 
-    func offer(_ remoteModel: MutationSync<AnyModel>) {
-        guard let queue = reconciliationQueues[remoteModel.model.modelName] else {
+    func offer(_ remoteModel: MutationSync<AnyModel>, isFullSync: Bool) {
+        guard var queue = reconciliationQueues[remoteModel.model.modelName] else {
             // TODO: Error handling
             return
         }
 
         queue.enqueue(remoteModel)
+        queue.isFullSync = isFullSync
```

Comment:
This is the wrong level to be tracking `isFullSync`. An incoming reconciliation queue processes individual items, and shouldn't know or care whether those items are coming from a full sync, a delta sync, or an individual subscription update.

Reply:
the `queue` here is of type `AWSModelReconciliationQueue`, I don't think the `AWSIncomingEventReconciliationQueue` class will hold this information